store.go

```diff
@@ -179,8 +183,12 @@ func (s *FilesystemStore) MaxLength(l int) {
 // Get returns a session for the given name after adding it to the registry.
 //
 // See CookieStore.Get().
-func (s *FilesystemStore) Get(r *http.Request, name string) (*Session, error) {
-	return GetRegistry(r).Get(s, name)
+func (s *FilesystemStore) Get(r *http.Request, name string) (*Session, *http.Request, error) {
```

Comment:
We shouldn't need to return the store here, if we're already replacing the `*http.Request` passed in.